packages/nimble-components/src/table-column/menu-button/cell-view/index.ts

```diff
@@ -0,0 +1,87 @@
+import { DesignSystem } from '@microsoft/fast-foundation';
+import { observable } from '@microsoft/fast-element';
+import { TableCellView } from '../../base/cell-view';
+import type {
+    TableColumnMenuButtonCellRecord,
+    TableColumnMenuButtonColumnConfig
+} from '..';
+import { template } from './templates';
+import { styles } from './styles';
+import type { MenuButton } from '../../../menu-button';
+import type { MenuButtonToggleEventDetail } from '../../../menu-button/types';
+import type { CellViewSlotRequestEventDetail } from '../../../table/types';
+import { cellViewMenuSlotName } from '../types';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-table-column-menu-button-cell-view': TableColumnMenuButtonCellView;
+    }
+}
+
+/**
+ * The cell view base class for displaying a string field as a menu button.
+ */
+export class TableColumnMenuButtonCellView extends TableCellView<
+TableColumnMenuButtonCellRecord,
+TableColumnMenuButtonColumnConfig
+> {
+    /** @internal */
+    public menuButton?: MenuButton;
+
+    /** @internal */
+    public valueSpan?: HTMLSpanElement;
+
+    /** @internal */
+    @observable
+    public hasOverflow = false;
+
+    public override get tabbableChildren(): HTMLElement[] {
+        if (this.menuButton) {
```

Comment:
Can you doublecheck what this'll do if the table starts out with non-empty string data, then it's replaced with empty strings? 
i.e. [this test](https://github.com/ni/nimble/blob/3165257cb2f224227c4f8dfa348a3dc090feb8ff/packages/nimble-components/src/table-column/menu-button/tests/table-column-menu-button.spec.ts#L109), but first set data as `[{ field: 'value' }]`, then set it to `[{ field: '' }]`

I think in that case, `this.menuButton` will be initialized but not in the DOM, and we don't want `tabbableChildren` to include it in that case.

(That's why the [anchor cell view checks this.showAnchor, not just this.anchor](https://github.com/ni/nimble/blob/297df53de72dc291959ee85bcdff240d96f3b4b3/packages/nimble-components/src/table-column/anchor/cell-view/index.ts#L68))

Reply:
Good catch! You were right that `this.menuButton` is still initialized (which seems wrong), but I've updated it as you suggested and added a test.